### DISCUSSION TITLE
Maxed out API allowance

### DISCUSSION THREAD

**Discussion by kme77** [nathanmarlor/foxess_em#81] [Q&A] [answered]
So I've somehow managed to max out my Solcast API limit by 4pm, the only reason I can think of is I was poking around in my install and restarted it several times (maybe over 10 times over a couple of hours)
In this scenario does the API get hit each time the integration restarts? If not then I'm not sure why I've been pounding the Solcast API?

Also, will the integration still function now? I assume not as all the predicted times show as 'unknown'.

Any ideas?

Ta 😊

**Comment by nathanmarlor** [comment, marked as answer]
I've been there a few times :)

The API limit resets at midnight UTC. If you're in the UK the integration will refresh the forecast five minutes before the off-peak period; for me this is 12:25 so everything starts working again.

**Reply by kme77** [reply]
Ahah, that's great news, thought I may have broken something by messing around where I don't belong, glad to hear that's not the case! 😋 
Thanks once again! 😁 👍
